filesystem/azure.go

```diff
@@ -212,7 +212,7 @@ func (f *AzureFile) Write(b []byte) (int, error) {
 		return 0, err
 	}
 	if err == io.EOF {
-		return 0, fmt.Errorf("Need blob content")
+		return 0, fmt.Errorf("Need write content")
```

Comment:
`need at least one full chunk size(%d) data, chunkSize`

Also can you explain why the first chunk is a special case (need it to be full)?